src/core/download/download.service.js

```diff
@@ -0,0 +1,12 @@
+export function download(csv, $window, $document) {
```

Comment:
put this file to core/services and rename to download.js,
$window, and $document can't be used in core, use window and document